tests/foreman/cli/test_leapp.py

```diff
@@ -0,0 +1,210 @@
+        f"--organization '{default_org.name}' --lifecycle-environment '{lc_env.name}'"
+    )
+    assert cmd_out.status == 0
+    assert ("AppStream RPMs 9" in cmd_out.stdout) and ("BaseOS RPMs 9" in cmd_out.stdout)
+
+    # Preupgrade conditions and check
+    rhel_contenthost.run("rm -rf /root/tmp_leapp_py3")
+    # Remove directory if in-place upgrade already performed from RHEL7 to RHEL8
+    rhel_old_ver = rhel_contenthost.run("cat /etc/redhat-release")
+    # 5. Update all packages and install Leapp utility
+    rhel_contenthost.run("yum clean all")
+    rhel_contenthost.run("yum repolist")
+
+    rhel_contenthost.run("subscription-manager release --set 8.8")
+    result = rhel_contenthost.run("dnf update -y")
+    assert result.status == 0
+    rhel_contenthost.run("yum install leapp-upgrade -y")
+
+    # Fixing inhibitors - download data files to avoid inhibitors
+    rhel_contenthost.run(
+        'curl -k "https://gitlab.cee.redhat.com/oamg/leapp-data/-/raw/stage/data/'
+        '{repomap.json,pes-events.json,device_driver_deprecation_data.json}"'
+        ' -o "/etc/leapp/files/#1"'
+    )
+    rhel_contenthost.run(
+        'sed -i "s/^AllowZoneDrifting=.*/AllowZoneDrifting=no/" /etc/firewalld/firewalld.conf'
+    )
+    # 6.1 Run LEAPP-PREUPGRADE Job Template-
+    template_id = (
+        target_sat.api.JobTemplate()
+        .search(query={'search': 'name="Run preupgrade via Leapp"'})[0]
+        .id
+    )
+    job = target_sat.api.JobInvocation().run(
+        synchronous=False,
+        data={
+            'job_template_id': template_id,
+            'targeting_type': 'static_query',
+            'search_query': f'name = {rhel_contenthost.hostname}',
+        },
+    )
+    target_sat.wait_for_tasks(
+        f'resource_type = JobInvocation and resource_id = {job["id"]}', poll_timeout=1800
+    )
+    result = target_sat.api.JobInvocation(id=job['id']).read()
+    assert result.succeeded == 1
+
+    # 6.2 Run LEAPP-UPGRADE Job Template-
+    template_id = (
+        target_sat.api.JobTemplate().search(query={'search': 'name="Run upgrade via Leapp"'})[0].id
+    )
+    job = target_sat.api.JobInvocation().run(
+        synchronous=False,
+        data={
+            'job_template_id': template_id,
+            'targeting_type': 'static_query',
+            'search_query': f'name = {rhel_contenthost.hostname}',
+            'inputs': {'Reboot': 'true'},
+        },
+    )
+    target_sat.wait_for_tasks(
+        f'resource_type = JobInvocation and resource_id = {job["id"]}', poll_timeout=1800
+    )
+    result = target_sat.api.JobInvocation(id=job['id']).read()
+    assert result.succeeded == 1
+    # Resume the rhel_contenthsot with ensure True to ping the virtual machine
+    rhel_contenthost.power_control(state=VmState.RUNNING, ensure=True)
+    rhel_new_ver = rhel_contenthost.run("cat /etc/redhat-release")
+    assert rhel_old_ver != rhel_new_ver
```

Comment:
Please update the assertion with the particular rhel version it should be updated to.

Reply:
@shweta83  What different assertions statement can be used instead of `assert old_ver != new_ver` condition?
ex-
`assert "9.2" in str(rhel_new_ver)`
`assert "Red Hat Enterprise Linux release 9.2 (Ootpa)" == str(rhel_new_ver)`